modules/nf-core/crisprcleanr/normalize/tests/main.nf.test

```diff
@@ -0,0 +1,68 @@
+
+nextflow_process {
+
+    name "Test Process CRISPRCLEANR_NORMALIZE"
+    script "../main.nf"
+    process "CRISPRCLEANR_NORMALIZE"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "crisprcleanr"
+    tag "crisprcleanr/normalize"
+
+    test("test-crisprcleanr-normalize") {
+
+        when {
+            process {
+                """
+                input[0] = [
+				    [ id:'test', single_end:false ], // meta map
+				    file(params.modules_testdata_base_path + 'genomics/homo_sapiens/genome/tsv/functional_genomics_counts.tsv', checkIfExists: true),
+				    file(params.modules_testdata_base_path + 'genomics/homo_sapiens/genome/tsv/library_functional_genomics.tsv', checkIfExists: true)
+				]
+				input[1] = 30
+				input[2] = 5
+
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(
+                    file(process.out.norm_count_file[0][1]).name,
```

Comment:
Do i understand it right that this only checks that the name matches? Why not the content of the file?

Reply:
Good point. I have now also added an assertion to verify that the file also contains 'sgRNA'.